src/wallet/coinselection.h

```diff
@@ -11,8 +11,8 @@
 #include <policy/feerate.h>
 #include <primitives/transaction.h>
 #include <random.h>
-#include <util/system.h>
 #include <util/check.h>
+#include <util/insert.h>
```

Comment:
Not strictly required, but I decided this makes sense, since these are currently in `namespace util` and are just a small amount of header-only code. See the discussion we had here: https://github.com/bitcoin/bitcoin/pull/27636#discussion_r1196623448.